### PR TITLE
Get and create vehicle years

### PR DESCRIPTION
**Pull request by rafaelrpbelo**
* Create vehicle's parser
* Create vehicle's catcher
* Create vehicle's impoter

### CODE REVIEW

File: lib/importer/vehicle_year.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+class Importer::VehicleYear
+  def initialize(reference, brand_id, vehicle_id, type)
+    @reference = reference
+    @brand_id = brand_id
+    @vehicle_id = vehicle_id
+    @type = type
+
+    @model = Models::VehicleYear
+    @catcher = Catcher::VehicleYear
+    @importer = Importer::VehicleYear
```
Comment (romulostorel) [inline lib/importer/vehicle_year.rb]: @rafaelrpbelo wtf

Comment (rafaelrpbelo) [inline lib/importer/vehicle_year.rb]: What?

Comment (romulostorel) [inline lib/importer/vehicle_year.rb]: @rafaelrpbelo this variable is getting the own class.

Comment (romulostorel) [inline lib/importer/vehicle_year.rb]: class Importer::VehicleYear
  @importer = Importer::VehicleYear 
end

**Pull request merged**